itest/litd_custom_channels_test.go

```diff
@@ -2768,3 +2769,432 @@ func testCustomChannelsFee(_ context.Context,
+	// final block which will confirm Alice's sweep transaction.
+	if len(sweepBlocks[0].Transactions) == 1 {
+		mineBlocks(t, net, 1, 1)
+	}
+
+	// With the sweep transaction confirmed, Alice's balance should have
+	// incremented by the amt of the HTLC.
+	aliceExpectedBalance += uint64(assetInvoiceAmt - 1)
+	assertSpendableBalance(
+		t.t, aliceTap, assetID, aliceExpectedBalance,
+	)
+
+	t.Logf("Mining enough blocks to time out the remaining HTLCs")
+
+	// At this point, we've swept two HTLCs: one from the remote commit, and one
+	// via the second layer. We'll now mine the remaining amount of blocks to
+	// timeout the HTLCs.
+	blockToMine := closeExpiryInfo.blockTillExpiry(
+		aliceHodlInvoices[1].preimage.Hash(),
+	)
+	mineBlocks(t, net, blockToMine, 0)
+
+	// We'll wait for both Alice and Bob to present their respective sweeps to
+	// the sweeper.
+	assertSweepExists(
+		t.t, alice,
+		walletrpc.WitnessType_TAPROOT_HTLC_LOCAL_OFFERED_TIMEOUT,
+	)
+	assertSweepExists(
+		t.t, bob,
+		walletrpc.WitnessType_TAPROOT_HTLC_OFFERED_REMOTE_TIMEOUT,
+	)
+
+	// We'll mine an extra block to trigger the sweeper.
+	mineBlocks(t, net, 1, 0)
+
+	t.Logf("Confirming initial HTLC timeout txns")
+
+	// Finally, we'll mine a single block to confirm them.
+	mineBlocks(t, net, 1, 2)
+
+	// At this point, Bob's balance should be incremented by an additional HTLC
+	// value.
+	bobExpectedBalance += uint64(assetInvoiceAmt - 1)
+	assertSpendableBalance(
+		t.t, bobTap, assetID, bobExpectedBalance,
+	)
+
+	t.Logf("Mining extra blocks for Alice's CSV to expire on 2nd level txn")
+
+	// Next, we'll mine 4 additional blocks to Alice's CSV delay expires for the
+	// second level timeout output.
+	mineBlocks(t, net, closeExpiryInfo.csvDelay, 0)
+
+	// Wait for Alice to extend the second level output to the sweeper before we
+	// mine the next block to the sweeper.
+	assertSweepExists(
+		t.t, alice,
+		walletrpc.WitnessType_TAPROOT_HTLC_OFFERED_TIMEOUT_SECOND_LEVEL,
+	)
+
+	t.Logf("Confirming Alice's final timeout sweep")
+
+	// With the way the sweeper works, we'll now need to mine an extra block to
+	// trigger the sweep.
+	sweepBlocks = mineBlocks(t, net, 1, 0)
+
+	// If the block mined above didn't also mine our sweep, then we'll mine one
+	// final block which will confirm Alice's sweep transaction.
+	if len(sweepBlocks[0].Transactions) == 1 {
+		// We'll mine one final block which will confirm Alice's sweep transaction.
+		mineBlocks(t, net, 1, 1)
+	}
+
+	// Finally, we'll assert that Alice's balance has been incremented by the
+	// timeout value.
+	aliceExpectedBalance += uint64(assetInvoiceAmt - 1)
+	assertSpendableBalance(
+		t.t, aliceTap, assetID, aliceExpectedBalance,
+	)
+
+	t.Logf("Sending all settled funds to Zane")
+
+	// As a final sanity check, both Alice and Bob should be able to send their
+	// entire balances to Zane, our 3rd party.
+	//
+	// We'll make two addrs for Zane, one for Alice, and one for bob.
+	zaneTap := newTapClient(t.t, zane)
```

Comment:
if we keep this we can remove the commented out zane tap init on top of the test

Reply:
Not sure what you mean. Zane is the universe server that information is posted to, then we use it at the very end to ensure that we can send the funds we've swept.